frontend/src/components/common/form/AutoCompleteText.test.tsx

```diff
@@ -28,6 +28,6 @@ describe('Auto Complete Text', () => {
         )}
       </Formik>,
     );
-    expect(pretty(container.innerHTML)).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Wouldn't a `firstChild` be different than `innerHTML`?  As the `innerHTML` could have many children.

Reply:
I thought the same thing, but if you look at the generated snapshots the contents match (minus some formatting changes).